workitem/workitem_repository.go

```diff
@@ -593,7 +593,7 @@ func (r *GormWorkItemRepository) Save(ctx context.Context, spaceID uuid.UUID, up
 		}
 		fieldValue := updatedWorkItem.Fields[fieldName]
 		var err error
-		if fieldName == SystemAssignees || fieldName == SystemLabels || fieldName == SystemBoardcolumns {
+		if fieldDef.Type.GetKind() == KindList {
```

Comment:
This change wasn't necessary for this PR but since this was a minor change I did it in this PR.